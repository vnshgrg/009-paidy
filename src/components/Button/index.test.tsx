import "react-native";
import { Text } from "react-native";
import React from "react";
import { Button } from "./index";

import renderer from "react-test-renderer";

it("renders Button correctly", () => {
  const tree = renderer
    .create(
      <Button>
        <Text>Test button</Text>
      </Button>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
