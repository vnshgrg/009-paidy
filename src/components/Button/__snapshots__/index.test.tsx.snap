// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Button correctly 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "alignItems": "center",
      "flexDirection": "row",
      "opacity": 1,
      "paddingHorizontal": 0,
      "paddingVertical": 0,
    }
  }
>
  <Text>
    Test button
  </Text>
</View>
`;
